### PR TITLE
Add touch note to button topics

### PR DESCRIPTION
**Pull request by ganicke**

### CODE REVIEW

**Comment by ganicke** [comment]
That would make common-packages docs depend on target content. Can I add a note about groundless operation?

**Comment by pelikhan** [comment]
You can add a note about capacitive touch vs resistive touch in the general sense.

**Comment by ganicke** [comment]
Oh, wait, I could extend the button docs to Adafruit and link the project. Do that?

File: libs/buttons/docs/reference/input/button/touch-sensors.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+(like your finger pushing on it) on its surface changes. The microprocessor is connected to this resistive
+stuff and knows when more or less electrical current flows through it. That's how it knows you touched it
+or lifted your finger away. Resistive touch material will work with other things too, not just you finger. A pen,
+pencil, or something with a pointed tip. If there is a large area of resistive touch material, like a
+computer display screen with a touch surface, the material can very accurately detect where you touched it
+anywhere on it's surface.
+
+## Capacitive touch
+
+A very popular way to detect touch is with capacitve touch sensors. Capacitive means using _capacitance_ which is about storing up a really, really small number of electrons of the surface of some material. When you touch the
```
Comment (pelikhan) [inline libs/buttons/docs/reference/input/button/touch-sensors.md]: capcitve

Comment (ganicke) [inline libs/buttons/docs/reference/input/button/touch-sensors.md]: Spelking is rong agin.

**Comment by ganicke** [comment]
Hmm, there's a different kind of 'resistive' touch. Maybe your thinking about the voltage divider deal with MaKey: https://learn.sparkfun.com/tutorials/makey-makey-quickstart-guide.

**Comment by pelikhan** [comment]
Yes makey makey is resistive so it needs to be grounded

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by ganicke** [comment]
This is cheating. They aren't 'real' touch sensors. It's just pulling a digital input low. This is could be done with an onPulsed() or looping on digitalRead().

**Pull request merged**